RoktDemo/Views/SubViews/DemoItemView.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  DemoItemView.swift
+//  RoktDemo
+//
+//  Copyright 2020 Rokt Pte Ltd
+//
+//  Licensed under the Rokt Software Development Kit (SDK) Terms of Use
+//  Version 2.0 (the "License");
+//
+//  You may not use this file except in compliance with the License.
+//
+//  You may obtain a copy of the License at https://rokt.com/sdk-license-2-0/
+
+
+import SwiftUI
+import Kingfisher
+
+struct DemoItemView: View {
+    let demoItemModel: DemoItemModel
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 10) {
+            HStack{
+                if demoItemModel.iconURL.isUrl() {
+                    KFImage.url(
+                        URL(string: demoItemModel.iconURL)
+                    )
+                    .resizable()
+                    .aspectRatio(contentMode: .fit)
+                    .frame(width: 100, height: 60, alignment: .leading)
+                } else if !demoItemModel.iconURL.isEmpty {
```

Comment:
what happens here if demoItemModel.iconURL is not empty, but invalid?

Reply:
I have tested it. It simply doesn't load the image. I think it is fine for this stage as we don't have any placeholder image to cover the failure images. What do you think?

Reply:
No worries, just making sure it didn't crash